Nodejs/Product/Npm/SPI/NodeModules.cs

```diff
@@ -125,13 +113,11 @@ private bool AddModuleIfNotExists(IRootPackage parent, string moduleDir, bool sh
 
             IPackage package = moduleInfo.Package;
 
-            if (package == null || depth == 1 || !moduleInfo.RequiredBy.Contains(parent.Path))
+            if (package == null || depth == 1)
```

Comment:
What was the purpose in looking into the path? Isn't this the logic that tells the user that a package is a dependency but has not been installed yet?

Reply:
no this seems to be more about being defensive about adding too many things to RequiredBy? It didn't seem to affect anything?